src/libs/actions/Report.js

```diff
@@ -481,91 +368,6 @@ function unsubscribeFromReportChannel(reportID) {
     Pusher.unsubscribe(pusherChannelName);
 }
 
-/**
- * Get the report ID for a chat report for a specific
- * set of participants and navigate to it if wanted.
- *
- * @param {String[]} participants
- * @param {Boolean} shouldNavigate
- * @returns {Promise<Object[]>}
- */
-function fetchOrCreateChatReport(participants, shouldNavigate = true) {
-    if (participants.length < 2) {
-        throw new Error('fetchOrCreateChatReport() must have at least two participants.');
-    }
-
-    return DeprecatedAPI.CreateChatReport({
-        emailList: participants.join(','),
-    })
```

Comment:
I think you can also delete this from the Deprecates API file

Reply:
And do a search for the function, because it's referenced in the Readme.md file

Reply:
I removed the actual method, but I think the readme is fine as an example even though it's gone. We're eventually going to run out of commands that are chained, but still want to tell contributors what _not_ to do so I think this is as good as any to leave